src/main/java/com/project/serverpool/service/ServerService.java

```diff
@@ -1,42 +1,56 @@
 package com.project.serverpool.service;
 
-import com.project.serverpool.database.AerospikeDatabase;
 import com.project.serverpool.domain.Client;
 import com.project.serverpool.domain.Server;
-import com.project.serverpool.model.dao.daoImpl.ClientDao;
-import com.project.serverpool.model.dao.daoImpl.ServerDao;
+import com.project.serverpool.repositories.ClientRepository;
+import com.project.serverpool.repositories.ServerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
 import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 @Service
 public class ServerService {
```

Comment:
You are not achieving abstraction , it is better to have an interface and have its methods implemented by the service.